aws/resource_aws_guardduty_member_test.go

```diff
@@ -152,7 +230,21 @@ resource "aws_guardduty_member" "test" {
 `, testAccGuardDutyDetectorConfig_basic1, accountID, email)
 }
 
-func testAccGuardDutyMemberConfig_invite(accountID, email, invitationMessage string) string {
+func testAccGuardDutyMemberConfig_invite(accountID, email string, invite bool) string {
+	return fmt.Sprintf(`
+%[1]s
+
+resource "aws_guardduty_member" "test" {
+  account_id                 = "%[2]s"
+  detector_id                = "${aws_guardduty_detector.test.id}"
+  disable_email_notification = true
+  email                      = "%[3]s"
+  invite                     = %[4]t
```

Comment:
by taking this approach (`%[1]s` vs `%s`) golint won't be able to determine if the types are wrong

Reply:
I really want to fix this upstream 😄

We should probably decide if we want to holistically go through all the test configurations and pick a format string style (indexed/unindexed) or use TF variables. Let's chat.